c/tskit/core.h

```diff
@@ -528,21 +528,25 @@ The table collection was not indexed.
 */
 #define TSK_ERR_TABLES_NOT_INDEXED                                  -702
 /**
+There was an error with the table's indexes.
+*/
+#define TSK_ERR_TABLES_BAD_INDEXES                                  -703
+/**
 Tables cannot be larger than 2**31 rows.
 */
-#define TSK_ERR_TABLE_OVERFLOW                                      -703
```

Comment:
We can't make these kinds of changes post 1.0. Just put the new error at the end of the table as 707